ruby/lib/ci/queue/redis/build_record.rb

```diff
@@ -59,7 +59,7 @@ def record_success(id, stats: nil)
 
         def record_flaky(id, stats: nil)
           redis.pipelined do |pipeline|
-            pipeline.sadd(
```

Comment:
I do not see `sadd` deprecated at https://www.rubydoc.info/gems/redis/Redis%2FCommands%2FSets:sadd though